src/Features/CSharp/Portable/EditAndContinue/SyntaxUtilities.cs

```diff
@@ -83,7 +83,7 @@ public static void AssertIsBody(SyntaxNode syntax, bool allowLambda)
             if (LambdaUtilities.IsLambdaBody(syntax))
             {
                 Debug.Assert(allowLambda);
-                Debug.Assert(syntax is ExpressionSyntax || syntax is BlockSyntax);
+                Debug.Assert(syntax is ExpressionSyntax || syntax is BlockSyntax || syntax is ArrowExpressionClauseSyntax);
```

Comment:
When I made changes for extracting bodies from local functions, I understood the meaning of the comment. :-) Thank you!

---
In reply to: [132499185](https://github.com/dotnet/roslyn/pull/21396#discussion_r132499185) [](ancestors = 132499185)